packages/legend-studio/src/stores/StudioPlugin.ts

```diff
@@ -214,4 +221,14 @@ export interface DSL_StudioPlugin_Extension extends StudioPlugin {
    * Get the list of the supported drag-and-drop type speficiers for grammar text editor.
    */
   getExtraGrammarTextEditorDnDTypes?(): string[];
+
+  /**
+   * Get the list of the supported visualize file content button.
+   */
+  getVisualizeFileContentButtons?(): ExtraFileGenerationtButton[];
```

Comment:
Please merge these (see comment above for the name, i.e. `getExtraFileGenerationResultViewerActions`) and name them as well as update the documentation accordingly

Reply:
Fixed! Merged the two interfaces with proper naming.